github/teams_test.go

```diff
@@ -463,3 +463,92 @@ func TestTeamsService_ListPendingTeamInvitations(t *testing.T) {
 		t.Errorf("Teams.ListPendingTeamInvitations returned %+v, want %+v", invitations, want)
 	}
 }
+
+func TestTeamsService_ListProjects(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	acceptHeaders := []string{mediaTypeNestedTeamsPreview, mediaTypeProjectsPreview}
+	mux.HandleFunc("/teams/1/projects", func(w http.ResponseWriter, r *http.Request) {
+		testMethod(t, r, "GET")
+		testHeader(t, r, "Accept", strings.Join(acceptHeaders, ", "))
+		fmt.Fprint(w, `[{"id":1}]`)
+	})
+
+	projects, _, err := client.Teams.ListTeamProjects(context.Background(), 1)
+	if err != nil {
+		t.Errorf("Teams.ListTeamProjects returned error: %v", err)
+	}
+
+	want := []*Project{{ID: Int64(1)}}
+	if !reflect.DeepEqual(projects, want) {
+		t.Errorf("Teams.ListTeamProjects returned %+v, want %+v", projects, want)
+	}
+}
+
+func TestTeamsService_ReviewProjects(t *testing.T) {
+	client, mux, _, teardown := setup()
+	defer teardown()
+
+	acceptHeaders := []string{mediaTypeNestedTeamsPreview, mediaTypeProjectsPreview}
```

Comment:
Would it make sense to rename `acceptHeaders`, to `wantHeaders`? Might help future readers know it's intended use.

Reply:
I like that idea for `*_test.go` files, @gauntface.

Looking through the code:

```bash
$ grep acceptHeaders github/*_test.go | wc -l
      38
$ grep wantHeaders github/*_test.go | wc -l
       0
```

we have 38 uses of `acceptHeaders` in tests currently and no `wantHeaders`.

For consistency, let's leave this one as-is, and I will open up a new issue that someone can switch all the tests to `wantHeaders`.